boards/nucleo-wl55jc/include/periph_conf.h

```diff
@@ -85,6 +85,8 @@ static const uart_conf_t uart_config[] = {
 static const spi_conf_t spi_config[] = {
     {
         .dev      = SPI1,
+/* SUBGHZ DEBUG PINS use the SPI1 pins */
+#if !IS_ACTIVE(CONFIG_WL55JC_SUBGHZ_DEBUG)
```

Comment:
does this prevent the usage of SPI1? If so, the whole entry should be removed

Reply:
Done